config/example/config.toml

```diff
@@ -67,64 +67,64 @@ enable_localhost_autoadmin = true
 # +MAINTAINER = Access to super dangerous debug verbs that can break an entire round. Use sparingly.
 # +DEV_TEAM = Access to dev team chat
 admin_ranks = [
-  # Format: {name = "My Rank", rights = ["+LIST", "+OF", "+RIGHTS"]}
-  {name = "Mentor", rights = [
-    "+MENTOR",
-  ]},
-  {name = "Trial Admin", rights = [
-    "+ADMIN",
-    "+BAN",
-    "+STEALTH",
-    "+REJUVINATE",
-  ]},
-  {name = "Game Admin", rights = [
-    "+ADMIN",
-    "+BAN",
-    "+STEALTH",
-    "+REJUVINATE",
-    "+DEBUG",
-    "+BUILDMODE",
-    "+EVENT",
-    "+SERVER",
-    "+POSSESS",
-    "+PROCCALL",
-    "+VAREDIT",
-    "+SOUND",
-    "+SPAWN",
-  ]},
-  {name = "Head of Staff", rights = [
-    "+EVERYTHING",
-  ]},
-  {name = "Hosting Provider", rights = [
-    "+EVERYTHING",
-  ]},
-  {name = "Maintainers", rights = [
-    "+EVERYTHING",
-  ]},
-  {name = "Developer", rights = [
-    "+DEV_TEAM",
-  ]},
+	# Format: {name = "My Rank", rights = ["+LIST", "+OF", "+RIGHTS"]}
```

Comment:
Looks like you were hit by auto-format on save -- please revert the extraneous changes to this file.

(It's annoying as hell, I know)

Reply:
Gah, fixed